public/jsx/components/product-detail.jsx

```diff
@@ -25,7 +25,7 @@ module.exports = React.createClass({
     // l10n. Also let's add a localized currency symbol.
     return (
       <div className="product">
-        <h1 className="title">{productData.id}</h1>
+        <h1 className="title">{productData.description.en}</h1>
```

Comment:
It seems to me like we're missing something from the config. Don't we want a shorter product name here? A "title" ? A description sounds like it could be a brief summary but maybe the term "description" is misleading.